cypress/integration-qe/e2e-networkValidation.spec.js

```diff
@@ -0,0 +1,158 @@
+import {
+    CLUSTER_NAME,
+    API_VIP,
+    INGRESS_VIP,
+    DNS_DOMAIN_NAME,
+    openCluster,
+    saveClusterDetails
+} from './shared';
+  
+import {
+    VALIDATE_CHANGES_TIMEOUT,
+} from './constants';
+
+const VIP_OUT_CIDR = '1.1.1.1'
+const MASTER_IP = ''
+const nicsTableCell = (row, label) =>
```

Comment:
This identifier can be in the shared file

Reply:
fixed